File: Python/1060 - Positive Numbers.py
n1,n2,n3,n4,n5,n6 = float(input()),float(input()),float(input()),float(input()),float(input()),float(input())
l1 = [n1,n2,n3,n4,n5,n6]
counter = 0
for i in range(len(l1)):
    if l1[i] > 0:
        counter += 1
print("%d valores positivos" % counter)

File: C++/1060 - Positive Numbers.cpp
#include <iostream>

int main()
{
    double num;
    int count = 0;
    for(int i = 0; i < 6; i++)
    {
        std::cin >> num;
        if(num > 0)
        {
            count ++;
        }
    }
    printf("%d valores positivos\n",count);
    return 0;
}